payroll-processor-client/src/app/app.module.ts

```diff
@@ -1,37 +1,42 @@
-import { BrowserModule } from '@angular/platform-browser';
+import { HttpClientModule } from '@angular/common/http';
 import { NgModule } from '@angular/core';
-
-import { AppRoutingModule } from './app-routing.module';
-import { AppComponent } from './app.component';
+import { FormsModule, ReactiveFormsModule } from '@angular/forms';
+import { BrowserModule } from '@angular/platform-browser';
 import { BrowserAnimationsModule } from '@angular/platform-browser/animations';
-import { RiskListComponent } from './risk-list/risk-list.component';
 
+import { FontAwesomeModule } from '@fortawesome/angular-fontawesome';
 import { NgbModule } from '@ng-bootstrap/ng-bootstrap';
+
+import { AboutComponent } from './about/about.component';
+import { AppRoutingModule } from './app-routing.module';
+import { AppComponent } from './app.component';
+import { EmployeeCreateComponent } from './employee/employee-create/employee-create.component';
 import { EmployeeListComponent } from './employee/employee-list/employee-list.component';
-import { HttpClientModule } from '@angular/common/http';
 import { EmployeeComponent } from './employee/employee.component';
-import { PayrollComponent } from './payroll/payroll.component';
-import { FontAwesomeModule } from '@fortawesome/angular-fontawesome';
-import { AboutComponent } from './about/about.component';
 import { PayrollListComponent } from './payroll/payroll-list/payroll-list.component';
+import { PayrollComponent } from './payroll/payroll.component';
+import { RiskListComponent } from './risk-list/risk-list.component';
 
 @NgModule({
   declarations: [
```

Comment:
Two things we can start to look at doing, as this App Module grows:

1. SCAM ([Single Component Angular Modules](https://medium.com/marmicode/your-angular-module-is-a-scam-b4136ca3917b)) - this is a nice way to isolate and clarify what depends on what in an Angular app as it grows.

We'll end up with an Angular module definition for each component, and anywhere we want to re-use that component (eg another component's template), we will import that module. This will bring along all of its dependencies and keep the module definitions readable.

2. [Lazy modules for app features](https://angular.io/guide/lazy-loading-ngmodules).

In the routing definition, we'll use the dynamic import syntax (`import('...').then(...)`) to help Angular split code up into multiple ES Modules.

This will lower the initial payload cost of the app and help define domain boundaries in the app.